src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -335,13 +343,40 @@ Demos
 
 ## FAQ
```

Comment:
Avoid FAQs. If it's new how do you know the "questions" are frequently asked. The content in FAQs really needs to be give a proper place in the broader document structure. If these questions are really asked, consider putting them on stack overflow.